nix/sources.json

```
@@ -29,10 +29,10 @@
         "homepage": "https://input-output-hk.github.io/haskell.nix",
         "owner": "input-output-hk",
         "repo": "haskell.nix",
-        "rev": "ded93b38ff656d3bf1b90f926bbf0ecea3fab723",
-        "sha256": "12pw3ykv2wp6kj0vj8y499p4khmzyx9r9il15l8p2wr2i7cpjxnc",
+        "rev": "6c57d4f978867a69414362f6b5da64e4afbee32b",
+        "sha256": "1x8iif585hhyh0ilj65rv3qxb7ff86pj58x8dh723wi4hbbsji5s",
```

```suggestion
        "rev": "f38d6c20b37df4a5a36817b18353c1734a303675",
        "sha256": "1vyxw0lh314lpxhiagllwl6pd2a9i9yclpiz8ks7s5xy87wcnrsf",
```
Take the latest rev as of today.